sbin/cubeit

```diff
@@ -527,9 +549,12 @@ IFS=$OLDIFS
 	# puppet modules etc.
 	if [ -v INSTALL_PUPPET_DIR ]; then
 	    if [ -d ${INSTALLER_FILES_DIR}/${INSTALL_PUPPET_DIR} ]; then
-		debugmsg ${DEBUG_INFO} "Copying Puppet files to install media"
-		recursive_mkdir ${mnt_rootfs}${INSTALLER_TARGET_FILES_DIR}/puppet
-		cp -r ${INSTALLER_FILES_DIR}/${INSTALL_PUPPET_DIR}/* ${mnt_rootfs}${INSTALLER_TARGET_FILES_DIR}/puppet/
+			INSTALL_PUPPET_DIR=${INSTALLER_FILES_DIR}/${INSTALL_PUPPET_DIR}
+		fi
+		if [ -d ${INSTALL_PUPPET_DIR} ]; then
```

Comment:
If $INSTALL_PUPPET_DIR is configured by interactive configure process, it will not be found in the ${INSTALLER_FILES_DIR}/${INSTALL_PUPPET_DIR}. So here we make sure that we don't depend on this INSTALLER_FILES_DIR variable.